server/controllers/messageController.js

```diff
@@ -18,16 +18,24 @@ exports.create = async (req, res, next) => {
 
 exports.read = async (req, res, next) => {
   try {
-    const page = 30 * req.query.page;
+    const limit = req.query.size * 1;
```

Comment:
I was thinking more along the lines of having a constant defined externally. This is still fine, but we might still want to have a fallback value in case a limit isn't provided.

Reply:
Ahh yeah that makes sense too haha, great suggestion though! Have added fallbacks for both queries.